tests/components/doorbird/test_device.py

```diff
@@ -12,6 +13,91 @@
 from .conftest import DoorbirdMockerType
 
 
+@pytest.fixture
+def doorbird_favorites_with_duplicates() -> dict[str, dict[str, Any]]:
+    """Return favorites fixture with duplicate events.
+
+    Creates favorites where identifiers "0" and "2" both have the same event name
+    (mydoorbird_doorbell), and both are referenced in the schedule.
+    """
+    return {
+        "http": {
+            "0": {
+                "title": "Home Assistant (mydoorbird_doorbell)",
+                "value": "http://127.0.0.1:8123/api/doorbird/mydoorbird_doorbell?token=TOKEN1",
+            },
+            # Duplicate favorite with same event name - both "0" and "2" will be
+            # in the schedule as doorbell outputs
+            "2": {
+                "title": "Home Assistant (mydoorbird_doorbell)",
+                "value": "http://127.0.0.1:8123/api/doorbird/mydoorbird_doorbell?token=TOKEN2",
+            },
```

Comment:
This seems like a bug that we wouldn't be cleaning up old urls with a different token